shared/common-adapters/input.native.js

```diff
@@ -13,38 +13,24 @@ import type {Props} from './input'
 type State = {
   focused: boolean,
   height: ?number,
-  value: string,
-  selections: {selectionStart: number, selectionEnd: number},
+  selections: {selectionStart: number, selectionEnd: number} | null,
 }
 
 class Input extends Component<Props, State> {
   state: State
-  _input: any
+  _input: NativeTextInput | null
 
   constructor(props: Props) {
     super(props)
 
+    // For now, leave selections null until we get the first selection
+    // notification, since we're unable to set selection on
+    // NativeTextInput.
+
     this.state = {
       focused: false,
       height: null,
-      value: props.value || '',
-      selections: {selectionStart: 0, selectionEnd: 0},
-    }
-  }
-
-  setNativeProps(props: Object) {
```

Comment:
Did you try this everywhere? Without this you can crash out in clickable box wo a wrapping box

Reply:
Thanks, I didn't know about `setNativeProps` being magic! Brought it back, added comment.